core/trino-main/src/main/java/io/trino/metadata/DisabledSystemSecurityMetadata.java

```diff
@@ -195,6 +195,15 @@ public Optional<Identity> getFunctionRunAsIdentity(Session session, CatalogSchem
         return Optional.empty();
     }
 
+    @Override
+    public void setFunctionOwner(Session session, CatalogSchemaFunctionName function, TrinoPrincipal principal)
+    {
+        throw notSupportedException(function.getCatalogName());
+    }
+
+    @Override
+    public void functionCreated(Session session, CatalogSchemaFunctionName function, TrinoPrincipal principal) {}
```

Comment:
What's `principal` for? Isn't `session` enough to get the identity of the creator of the function?